### PR TITLE
Add title-case filter

### PR DESCRIPTION
**Pull request by lcrossan**
https://github.com/a8m/angular-filter/issues/238

;)

### CODE REVIEW

File: src/_filter/string/title-case.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/**
+ * @ngdoc filter
+ * @name title-case
+ * @kind function
+ *
+ * @description
+ * get string with {n} and replace match with enumeration values
+ */
+angular.module('a8m.title-case', [])
+  .filter('titleCase', function () {
```
Comment (a8m) [inline src/_filter/string/title-case.js]: What do you think about changing the name to `titleize`? I want this module to be aligned with [ng-pipes](https://github.com/a8m/ng-pipes/blob/master/src/string/titleize.pipe.ts)

File: src/_filter/string/title-case.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/**
+ * @ngdoc filter
+ * @name title-case
+ * @kind function
+ *
+ * @description
+ * get string with {n} and replace match with enumeration values
```
Comment (a8m) [inline src/_filter/string/title-case.js]: Can you please remove or replace this description?

**Comment by a8m** [comment]
Hi @lcrossan,
Thanks for the PR. I added some comments

**Comment by lcrossan** [comment]
sure thing! expect those changes tonight :)

**Comment by lcrossan** [comment]
just changed! thanks :)

**Comment by a8m** [comment]
That's great.
Thanks @lcrossan

**Comment by Anber** [comment]
Is it just alias for `ucfirst`?

**Comment by a8m** [comment]
Hey @Anber 
Thanks for your comment, I must to say that I only noticed last week that this filter is a duplicate.
I planned to remove these changes and add it as an alias like [here].(https://github.com/a8m/angular-filter/blob/master/src/_filter/collection/contains.js#L10-L13)

I want to help with that, feel free to make a PR.

**Pull request merged**